llvm/test/Transforms/LoopVectorize/reduction-small-size.ll

```diff
@@ -38,7 +38,7 @@ define i8 @PR34687(i1 %c, i32 %x, i32 %n) {
 ; CHECK-NEXT:    [[R:%.*]] = phi i32 [ [[BC_MERGE_RDX]], [[SCALAR_PH]] ], [ [[R_NEXT:%.*]], [[IF_END]] ]
 ; CHECK-NEXT:    br i1 [[C:%.*]], label [[IF_THEN:%.*]], label [[IF_END]]
 ; CHECK:       if.then:
-; CHECK-NEXT:    [[T0:%.*]] = sdiv i32 undef, undef
```

Comment:
This doesn't seem to impact the vector loop currently, because it is not used and will be removed presumably.

We could either remove it alltogether or add a user (and an unknown divisor) to make sure we need predication which seems to be the original intent of the test looking through https://github.com/llvm/llvm-project/issues/34035